### PR TITLE
Chapter 9 Validation

### PR DESCRIPTION
**Pull request by ghost**
- Some exercises in sec_conic_sections have no solution written.
- Some of the tabulars have weird text in them thich looks like a tex code segment spread across multiple cells. These were like this before the fix-dep script was ran over it.
- Like in previous chapters some of the solutions in webwork consist of a single image. Throws an error but a image is a solution so it shouldn't be an issue.
- sec_param_eqs has some images within a \<cell\> in a \<table\>. I don't see this as an issue but the validator doesn't like it. The images were wrapped in a \<sidebyside\> with one image so I stripped that out.
- Removed captions from some figures that were things like "(a)" since these figures would be given this information anyways. Figures are sometimes referenced like this where the parent figure is referenced but with the letter designation after. i.e. "In \<xref ref="fig_parcalc5"\>Figure\</xref\>(a) we see a plot of the second derivative." I believe ids should be added (as I tend to do, but sometimes they feel like they could be out of place). Such as the first child figure being "b" or the parent figure has the same id as the child (see the commented out figure ids in fig_parcalc5 in sec_par_calc.ptx I added IDs to the child figures that would make sense but you can clearly see how weird the old figure ids are).
- There are some very strange tabulars that will need a total redo. Something didn't convert correctly when this repo was constructed. These live between lines 1066 and 1510 in sec_polar so the errors within those lines have to do with these when validating.
- Deleted a fair bit of commented out image code since it was getting really messy and updated a few partially since they were simple.
- Also I noticed some tables with figures should perhaps be a \<sidebyside\>. I converted a few in sec_polarcalc.ptx.

Still needs testing.

### CODE REVIEW

**Comment by sean-fitzpatrick** [comment]
In sec_param_eqs the images in a table can be put into a sbsgroup.

**Comment by sean-fitzpatrick** [comment]
Which file had the tables that need fixing? you mentioned line numbers but not a file.
Regarding references to figures: yes, standard practice is to add xml:id for each image, like fig5a and fig5b, and then things are handled automatically.
There's a conflict due to a line added to .gitignore. Do you want to just revert that change before we attempt a merge?

**Comment by ghost** [comment]
Ahh yes I did forget to list the file name it but I have a commit on here that seems to fix it anyways with the sbsgroup.

**Comment by ghost** [comment]
Actually that was a different tabular I changed. The really weird ones which need to be updated still are in sec_polar around 1066 and 1510.  Looking back they just need to be places in sbs groups.

**Comment by sean-fitzpatrick** [comment]
General comment: at this point, you're well ahead of me. If I'm lucky I'll get to Chapter 9 before the summer runs out. Specific items of concern are maybe best done in a meeting. Next week, perhaps?

Specific thing: images in tables sounds wrong. I'd need to know more. There are cases where maybe you're using TikZ to make a small (character-sized) image for something you can't otherwise get off of a character map.
But if these are full size images (e.g. graphs) arranged into a table, probably sbsgroup is the way to handle them.

**Comment by sean-fitzpatrick** [comment]
Oh... I'm seeing the table in sec_param_eqs. Yes, this is bad. (As in, we shouldn't keep it as is.)
Where this came from: in the original LaTeX source, this was a margin table, stacked vertically, like so:

|-----------|
|table stuff|
     (a)
<image>
     (b)

In PreTeXt this should become a sidebyside, with a tabular and an image.

**Comment by ghost** [comment]
Probably, I did fix the huge broken tabular for the polar image gallery with figures and sbs groups a couple days ago so that is good now.

I also made that table with an image into a figure with a sidebyside containing 2 figures where one is a tabular and the other is an image and it appears to look correct. 

Also whenever you are free after Monday we can get together for a meeting.

File: src_pp/sec_conic_sections.ptx [hunk 1]
```diff
@@ -19,23 +18,6 @@
...
-        &amp;
-        &amp;
-      </figure>
-
-      <figure>
-        <caption>Point&amp;  Line &amp;  Crossed Lines</caption>
-        &amp;
-        &amp;
-      </figure>
-    </sidebyside>
```
Comment (sean-fitzpatrick) [inline src_pp/sec_conic_sections.ptx:38]: This figure shouldn't be deleted. But it probably should be done as an sbsgroup with a row of 4 followed by a row of 3.
There are png files that go with this figure.

**Review by sean-fitzpatrick — CHANGES_REQUESTED** [review]
This all looks good. Nice work on the gallery of polar curves. I haven't built the HTML yet to check the output but it's a good start.
Only change needed is the one in the conic sections section: that first figure with 7 images uses some PNG images rather than tikz.

File: src_pp/sec_conic_sections.ptx (hunk 1 above)
Comment (sean-fitzpatrick) [reply src_pp/sec_conic_sections.ptx:38]: Include the images via
`<image source="images/foo.jpg"/>`.
The files are in the original repo for the LaTeX code.
We'll need an `images` folder in output for both HTML and LaTeX.

**Comment by sean-fitzpatrick** [comment]
PNG files are now in place on the master branch.
They'll need to move at some point, but this gives us access.

**Pull request merged**